frontend/public/components/nav.jsx

```diff
@@ -219,12 +247,28 @@ const NavSection = connect(navSectionStateToProps)(
       // we could use scaleY, but that literally scales along the Y axis, ie shrinks
       // we could use flexbox or the equivalent to get an actual height, but this is the easiest solution :-/
 
-      const maxHeight = !this.state.isOpen ? 0 : 29 * _.get(this.props.children, 'length', 1);
+      const childrenArray = React.Children.toArray(children);
+      let addSeparator = true;
+      if (!_.isEmpty(_.get(this,['props', 'promotedCrds']))){
+        _.each(this.props.promotedCrds, (ce) => {
+          if (ce.spec.navExtension.navSection === this.props.text) {
+            if (addSeparator) {
+              childrenArray.push(<Sep />);
+              addSeparator = false;
+            }
+            const link = ce.spec.scope === 'Namespaced'
+              ? formatNamespacedRouteForCrd(ce.spec.reference, activeNamespace)
+              : `/k8s/cluster/${ce.spec.reference}`;
```

Comment:
For both of these links, I think we should be able to use existing utilities for getting the route